src/app/_models/targets.ts

```diff
@@ -0,0 +1,37 @@
+import * as am4charts from '@amcharts/amcharts4/charts';
+
+export enum TargetFieldName {
+  THREE_D = 'three_d',
+  HQ = 'hq',
+  TOTAL = 'total'
```

Comment:
Same here, perhaps `totalNumberRecords`?

Reply:
I think that's redundant (and would require other names to also include the 'records' suffix) - the context makes the meaning clear.